sinon/test/TestSinonSpy.py

```diff
@@ -852,3 +856,18 @@ def test240_neverCalledWithMatch_args(self):
         self.assertTrue(spy.neverCalledWithMatch("a", "e"))
         self.assertTrue(spy.neverCalledWithMatch("d", "e", "c")) #it's a combination
 
+    @sinontest
+    def test270_args(self):
+        spy = SinonSpy(E_func)
+        sinon.g.E_func()
+        self.assertListEqual(spy.args, [()])
+        self.assertListEqual(spy.kwargs, [{}])
+        sinon.g.E_func(1, a=1)
+        self.assertListEqual(spy.args, [(), (1,)])
+        self.assertListEqual(spy.kwargs, [{}, {'a':1}])
+        sinon.g.E_func(1, 2, a=1, b=2)
+        self.assertListEqual(spy.args, [(), (1,), (1,2)])
+        self.assertListEqual(spy.kwargs, [{}, {'a':1}, {'a':1,'b':2}])
+
+if __name__ == "__main__":
```

Comment:
Since current CI uses `python -m unittest <target_test>` to run test. Before pushing the code, you should remove this part.

Reply:
I find that the `if __name__...` code makes it easier to unit test a single test set at a time, but it's your code base so I'll remove it. :)